website/content/en/status/report-2024-01-2024-03/packagekit.adoc

```diff
@@ -0,0 +1,12 @@
+=== PackageKit backend for FreeBSD pkg
+
+Contact: Gleb Popov <arrowd@FreeBSD.org>
+
+PackageKit is a small D-Bus daemon program that serves as a backend for "application store" type of apps - most notably Plasma Discover and Gnome Software Center.
+The latest PackageKit release features a libpkg backend, which means that you can now use PackageKit-enabled programs on FreeBSD to manage software.
+Plasma Discover is already switched to using PackageKit, so you will get it working out of the box once you update your ports/packages.
+
+If you observe any crashes or bugs in PackageKit please let me know by opening an issue link:https://github.com/PackageKit/PackageKit/issues[upstream].
```

Comment:
I'd move "opening an issue" into the link text (`by link:https://github.com/PackageKit/PackageKit/issues[opening an issue upstream]`) fir better accessibility. Other than that, it's good to go.